EIPS/eip-3475.md

```diff
@@ -357,65 +454,107 @@ e.g.
 
 
 ## Backwards Compatibility
 
-ERC-3475 contract is not compatible with contracts that don't have an ERC-3475 interface built in. This requires the existing contract to upgrade with ERC-3475 interface. The receiving of ERC-3475 Bond need the implementation of ERC-3475 interface in the receiver contract.
+ERC-3475 contract is  compatible with contracts that  inherit  ERC-3475 interface .Its valid for both the issuer and receiver of the bonds , the client EOA needs the IERC3475 interface.
 
-However any existing ERC-20 token contract can issue their ERC-3475 bond, by giving the minting role to a bank contract with ERC-3475 interface built in. The implementation of This can be found in our Use Cases.
+However any existing ERC-20 token contract can issue their ERC-3475 bond, by delegating the  the minting role to a bank : orchestrator contract with ERC-3475 interface built in. The implementation of This can be found in our reference implementation.
 
 To ensure the reading of transactions, `"Issued"` ,`"Redeemed"` ,`"Burned"` ,`"Transfer"`, Events cited above MUST be emitted when such transaction is passed. 
 
-***Note that the ERC 3475 interface is also compatible with ERC-20 and [ERC-721](./eip-721.md) interface .*** But the creation of a separated bank contract is recommended for reading and future upgrade needs and .
+***Note that the ERC 3475 interface is also compatible with ERC-20 and [ERC-721](./eip-721.md) interface .*** But the creation of a separated bank contract is recommended for reading and future upgrade needs .
 
 The issuing of ERC-3475 bonds is not limited to ERC-20 token. Standard like ERC-721 nonfungible token can also issue their bond with the help of ERC-3475 interface. 
 
-Any ERC3475 bond or any hybrid of ERC-20 and ERC-721 contract can be used as the collateral for another ERC-3475 bond. This allows the market to create bond represent no longer a single type of ERC-20 token. Bond can now represent a collection of collaterals in ERC-20 token, ERC-721 nonfungible token and ERC-3475 Multiple callable bonds...
+Any ERC3475 bond or any hybrid of ERC-20 and ERC-721 contract can be used as the collateral for  an given ERC-3475 bond. This allows the market to create bond represent no longer a single type of ERC-20 token. Bond can now represent a collection of collaterals in ERC-20 token, ERC-721 nonfungible token and ERC-3475 Multiple callable bonds...
```

Comment:
```suggestion
Acceptable collateral can be in the form of [EIP-20](./eip-20.md) tokens, [EIP-721](./eip-721.md) tokens, or other bonds.
```

Would also recommend including EIP-1155 in this list. Not sure if this is meant to be in the backwards compatibility section though.